### PR TITLE
Remove emotion

### PR DESCRIPTION
**Pull request by ddonahue007**
initial starting point of removing "emotion" in favor of sass styling. 

Work has been done for the following directory
- components/reports/reportSummary

### CODE REVIEW

File: src/components/reports/reportSummary/reportSummaryAlt.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+.container {
+  display: flex;
+}
+
+.cost {
+  flex-Grow: 1;
+  min-Height: 470px;
+  margin-Right: var(--pf-global--spacer--md);
```
Comment (dlabrecq) [inline src/components/reports/reportSummary/reportSummaryAlt.scss:16]: I believe the properties should all be lower case (e.g.,  margin-right).

File: src/components/reports/reportSummary/reportSummaryDetails.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+.reportSummaryDetails {
+    margin-Bottom: var(--pf-global--spacer--md);
+    display: flex;
+    align-Items: flex-end;
+}
+
+.text {
+  padding-Bottom: var(--pf-global--spacer--sm);
+  line-Height: var(--pf-global--LineHeight--sm);
+  font-Size: var(--pf-global--icon--FontSize--sm);
```
Comment (dlabrecq) [inline src/components/reports/reportSummary/reportSummaryDetails.scss:12]: I believe this font size should be xs.

**Review by dlabrecq — COMMENTED** [review]
This is looking good. Although, I noticed a bunch of new warnings in the build.

Screen Shot 2020-09-15 at 3 21 02 PM

File: src/components/reports/reportSummary/reportSummaryDetails.scss (hunk 2 above)
Comment (ddonahue007) [reply src/components/reports/reportSummary/reportSummaryDetails.scss]: fixed

File: src/components/reports/reportSummary/reportSummaryAlt.scss (hunk 1 above)
Comment (ddonahue007) [reply src/components/reports/reportSummary/reportSummaryAlt.scss]: fixed

**Review by dlabrecq — APPROVED** [review]
Looks great. Thanks for the contribution!

**Review by boaz0 — COMMENTED** [review]
LGTM :+1:

File: src/components/reports/reportSummary/reportSummaryDetails.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  padding-bottom: var(--pf-global--spacer--sm);
+  line-height: var(--pf-global--LineHeight--sm);
+  font-size: var(--pf-global--icon--FontSize--xs);
+}
+
+.units {
+  padding-left: var(--pf-global--spacer--xs);
+  padding-bottom: var(--pf-global--spacer--sm);
+  line-height: var(--pf-global--LineHeight--sm);
+  font-size: var(--pf-global--icon--FontSize--xs);
```
Comment (dlabrecq) [inline src/components/reports/reportSummary/reportSummaryDetails.scss:19]: Discovered that these font styles are not valid. Our hero numbers are no longer styled.

File: src/components/reports/reportSummary/reportSummaryDetails.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  white-space: nowrap,
+}
+
+.value {
+  color: var(--pf-global--Color--100);
+  margin-right: var(--pf-global--spacer--sm);
+  font-size: var(--pf-global--icon--FontSize--4xl);
```
Comment (dlabrecq) [inline src/components/reports/reportSummary/reportSummaryDetails.scss:26]: Discovered that these font styles are not valid. Our hero numbers are no longer styled.

**Pull request merged**